ui/src/store.js

```diff
@@ -10,6 +10,8 @@ export const store = new Vuex.Store({
       username: VueCookies.get("username") || null,
       userId: VueCookies.get("userId") || null
     },
+    roles: VueCookies.get("roles") || null,
+    isAllowToAccessTheNextPage:false,
```

Comment:
Remove is from the variable name. Is is typically used for a method name.

Reply:
Done